Gemfile

```diff
@@ -6,7 +6,7 @@ gem "capybara-select2"
 gem "docker-api", "~> 1.34"
 gem "docker-compose", "~> 1.1"
 gem "faker", "~> 1.9"
-gem "govuk-lint", "~> 3.11.2"
+gem "govuk-lint", "4.0.0"
```

Comment:
This should be set `"~> 4.0"` to be clearer that we expect to be compatible with all minor versions of 4.

Reply:
Thanks. Done!